### ISSUE TITLE
polyserve - does not run

### ISSUE THREAD

**Issue by rsORAU** [robdodson/chart-elements#32]
Is this supposed to run out of the box with a clean git clone using polyserve?  When I git clone this repo and run Polyserve, my browser produces a 404 error that it cannot find the 'http://localhost:8080/components/webcomponentsjs/webcomponents-lite.js".  I understand the owner of this repo is getting ready for IO 2016, so thank you in advance to anyone's feedback.

**Comment by robdodson** [comment]
Did you run bower install?

**Comment by rsORAU** [comment]
I just ran bower install, but unfortunately ran into the same error.  I do not know if this makes a difference, but I have to use Mozilla Firefox in my development environment.

**Comment by robdodson** [comment]
I just did a fresh checkout and installed polyserve fresh and it's working fine for me in firefox.

Steps to reproduce:

```
git clone https://github.com/robdodson/chart-elements.git
cd chart-elements
bower install
polyserve
```

Then open `http://localhost:8080/components/chart-elements/` in your browser`

**Comment by rsORAU** [comment]
Thank you. Those are the steps that I followed. I am not sure if maybe it could then be something in my environment configuration, but I will try getting a fresh install of polyserve.

**Comment by rsORAU** [comment]
Thank you for your response. I followed the steps you provided, and installed polyserve while in the charts-elements directory, however I still received the same error when visiting the given URL in my browsers.  I have included a screenshot below. Since this is obviously not a problem with your project I will pull this question over to another more appropriate ticket.

![screenshot](https://cloud.githubusercontent.com/assets/19293942/15332624/5b9a663e-1c34-11e6-99f3-84402e1afeda.png)
